tests/test_gradient.py

```diff
@@ -139,12 +139,15 @@ def test_gradient_equivalence(self, shape, kernel, space_order, preset, nbl, dty
         assert(np.allclose(grad_u.data, grad_v.data, rtol=tolerance, atol=tolerance))
         assert(np.allclose(grad_u.data, grad_uv.data, rtol=tolerance, atol=tolerance))
 
-    @pytest.mark.parametrize('dtype', [np.float32, np.float64])
-    @pytest.mark.parametrize('space_order', [4])
-    @pytest.mark.parametrize('kernel', ['OT2'])
-    @pytest.mark.parametrize('shape', [(70, 80)])
-    @pytest.mark.parametrize('checkpointing', [True, False])
-    def test_gradientFWI(self, dtype, shape, kernel, space_order, checkpointing):
+    @pytest.mark.parametrize('dtype, space_order, kernel, shape, ckp, setup_func', [
+        (np.float32, 4, 'OT2', (70, 80), True, setup),
+        (np.float32, 4, 'OT2', (70, 80), False, setup),
+        (np.float64, 4, 'OT2', (70, 80), True, setup),
+        (np.float64, 4, 'OT2', (70, 80), False, setup),
+        (np.float64, 4, 'centered', (70, 80), True, tti_setup),
+        (np.float64, 4, 'centered', (70, 80), False, tti_setup),
+    ])
+    def test_gradientFWI(self, dtype, space_order, kernel, shape, ckp, setup_func):
         r"""
```

Comment:
is this a leftover?

Reply:
It seems so, and now I realize there are others in that file, thx.